### PR TITLE
Initial checkin

### LINKED ISSUE #99: TerriLLBean githubteacher Track Progress on GitHub Work Flow
# Terri's Workflow :palm_tree:

-[] create a branch (on my own branch)
-[] add commits
-[] open a pull request
-[] collaborate
-[] Make some more commits (on my own branch)
-[] merge our branch into master

### PR DESCRIPTION
**Pull request by TerriLLBean**
This pull request adds a file for Horley England.  @jackief  please review it. :hamster: 

This pull request resolves #99

### CODE REVIEW

**Comment by jackief** [comment]
I am going to try to hang out till the break before rebooting

File: TerriLLBean-horley-england.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Horley England
+### Restaurants
```
Comment (jackief) [inline TerriLLBean-horley-england.md:2]: I look forward to seeing this content!

File: TerriLLBean-horley-england.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+### Things to do
```
Comment (githubteacher) [inline TerriLLBean-horley-england.md]: I've never been to England, but I'd love to go! Are there any good outdoor activities?

File: TerriLLBean-horley-england.md (hunk 1 above)
Comment (TerriLLBean) [inline TerriLLBean-horley-england.md:2]: Me too!   My schedule is already planned out.. So I'll let you know!

Comment (kmmtechnology) [inline TerriLLBean-horley-england.md:2]: I am writing this comment just for testing ,collaborating and commenting purpose

File: TerriLLBean-horley-england.md (hunk 2 above)
Comment (TerriLLBean) [inline TerriLLBean-horley-england.md]: I will find out!  Going there in August!

**Pull request merged**